File: app/models/GeneralSettings.java
package models;

import models.base.GeneratedIdentityModel;

import javax.persistence.Entity;

@Entity
public class GeneralSettings extends GeneratedIdentityModel {

    private String name;

    private String value;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }
}

File: app/models/admin/GeneralSettings.scala
// SPDX-FileCopyrightText: 2024 The members of the EXAM Consortium
//
// SPDX-License-Identifier: EUPL-1.2

package models.admin

import jakarta.persistence.Entity
import models.base.GeneratedIdentityModel

import scala.compiletime.uninitialized

@Entity
class GeneralSettings extends GeneratedIdentityModel:
  var name: String  = uninitialized
  var value: String = uninitialized
